inventory/environ.py

```diff
@@ -183,6 +183,7 @@ def overrideEnvironmentVars(vars_scope):
         vars_scope["splunk"]["idxc"] = {}
     vars_scope["splunk"]["idxc"]["secret"] = os.environ.get('SPLUNK_IDXC_SECRET', vars_scope["splunk"]["idxc"]["secret"])
     vars_scope["splunk"]["enable_service"] = os.environ.get('SPLUNK_ENABLE_SERVICE', vars_scope["splunk"]["enable_service"])
+    vars_scope["splunk"]["build_location"] = os.environ.get('SPLUNK_INSTALLER', vars_scope["splunk"]["build_location"])
```

Comment:
This works and I think it's safe to leave it. But it should be noted in docs somewhere that `SPLUNK_INSTALLER` and `SPLUNK_BUILD_URL` will write to the same location `vars_scope["splunk"]["build_location"]`, with `SPLUNK_BUILD_URL` taking precedence.

Maybe we can also capture this behavior with the tests/small/test_environ.py